### PR TITLE
add target tracking CPU scaling policy

### PR DESCRIPTION
**Pull request by wjordan**
This PR replaces the [step scaling policy](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-simple-step.html) in our production (frontends-containing) CloudFormation stack with a [Target Tracking Scaling Policy](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-target-tracking.html), a new feature released [July 12 2017](https://aws.amazon.com/about-aws/whats-new/2017/07/introducing-target-tracking-scaling-policies-for-auto-scaling/) and the current best practice for dynamic scaling of an auto-scaling group based on average CPU load.

Basically it replaces our custom-defined scaling policy with a managed, AWS-internal implementation.

### CODE REVIEW

**Comment by wjordan** [comment]
I will apply this as an out-of-band stack update since our current scaling policies need to be fixed. I'll merge this PR once I've confirmed the template works correctly.

**Comment by wjordan** [comment]
Change has been applied and looks correct on production.
